cmd/query/main.go

```diff
@@ -79,12 +81,14 @@ func main() {
 				logger.Fatal("Failed to configure query service", zap.Error(err))
 			}
 
-			jt := jtracer.NoOp()
+			var tracerProvider trace.TracerProvider = nooptrace.NewTracerProvider()
+			var sdkTracerProvider *sdkTrace.TracerProvider
 			if queryOpts.EnableTracing {
-				jt, err = jtracer.New("jaeger-query")
```

Comment:
There's no need to change this line. Jtracer, once initialized, gives you TracerProvider, does it not?

Reply:
But, didn't you mention that we have to remove the jTracer wrapper itself?

Reply:
jTracer returns a wrapper that includes a TracerProvider along with a separate close function. SO, in order to pass trace.TracerProvider, I will have to pass jtracer.OTEL as an arg, that will disrupt several functions that are depending upon jTracer and not just the tracerProvider as evident in the server.go file (createHTTPServer and createGRPCServer)

Reply:
> jTracer returns a wrapper that includes a TracerProvider along with a separate close function. SO, in order to pass trace.TracerProvider, I will have to pass jtracer.OTEL as an arg, that will disrupt several functions that are depending upon jTracer and not just the tracerProvider as evident in the server.go file (createHTTPServer and createGRPCServer)

@yurishkuro Would appreciate your insights on this, should I figure out on how to change all of them?

Reply:
seems you figured it out? Current code looks right to me. Please resolve the thread if so.